troi.py

```diff
@@ -18,30 +17,39 @@ def cli():
 @click.argument('args', nargs=-1)
 @click.option('--debug', '-d', is_flag=True, default=False)
 def playlist(patch, args, debug):
-
+    """Generate a playlist using a patch"""
     if debug:
-        print("- debug mode on")
+        print("- debug mode on", file=sys.stderr)
 
     patches = troi.utils.discover_patches("patches")
-    if not patch in patches:
-        print("Cannot load patch '%s'. Use the list command to get a list of available patches." % patch)
-        quit()
```

Comment:
from my understanding, quit() should only be used in an interactive shell. Since these are error cases I uses sys.exit(1) to fail with an error code